apis/nucleus/src/components/Cell.jsx

```diff
@@ -372,7 +372,8 @@ const Cell = forwardRef(
       layout,
       halo.public.theme,
       halo.app,
-      themeName
+      themeName,
+      snOptions?.disableThemeBorder
```

Comment:
if possible I suggest converting this into object with props instead. Too many parameters makes it easy to pass them in the wrong order.